src/server.js

```diff
@@ -216,4 +266,6 @@ app.get(
 );
 
 const port = 8080;
-app.listen(port, () => console.log(`Server listening on port: ${port}`));
+app.listen(port, () =>
```

Comment:
Is there any reason this `console.log` was changed?

Reply:
Its the same thing, this basically logs a link to port 8080, instead of plain text